CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 2.1.1 (April 30, 2020)
 
+* Add input metadata for objects processing
 * Add the steward URL to the body of the outgoing message
 
 ## 2.1.0 (May 7, 2020)
```

Comment:
The update 2.1.1 should be a part of 2.1.0 (merged), because it was not published yet and it has the release date in future